src/backend/distributed/executor/multi_client_executor.c

```diff
@@ -396,7 +396,7 @@ MultiClientSendQuery(int32 connectionId, const char *query)
 	connection = ClientConnectionArray[connectionId];
 	Assert(connection != NULL);
 
-	querySent = PQsendQuery(connection->pgConn, query);
+	querySent = SendRemoteCommand(connection, query);
```

Comment:
we should also change the PQsendQuery in master_citus_tools.c::ExecuteCommandsInParallelAndStoreResults

Reply:
Fixed